traffic_ops/testing/api/v1/servers_test.go

```diff
@@ -16,9 +16,8 @@ package v1
 */
 
 import (
-	"testing"
-
 	"github.com/apache/trafficcontrol/lib/go-tc"
+	"testing"
```

Comment:
Why'd you re-order these? Normal procedure is to put standard library dependencies first, then in-project dependencies, then 3rd party.